Logstash/logstash-input-azureblob/lib/logstash/inputs/azureblob.rb

```diff
@@ -54,6 +54,17 @@ class LogStash::Inputs::LogstashInputAzureblob < LogStash::Inputs::Base
 
   # Set the container of the blobs.
   config :container, :validate => :string
+  
+  # The path(s) to the file(s) to use as an input.
+  # You can use filename patterns here, such as `logs/*.log`.
+  # If you use a pattern like `logs/**/*.log`, a recursive search
+  # of `logs` will be done for all `*.log` files.
+  # Do not include a leading `/`, as Azure path look like this:
+  # `path/to/blob/file.txt`
+  #
+  # You may also configure multiple paths. See an example
+  # on the <<array,Logstash configuration page>>.
+  config :path, :validate => :array, :default => ["**/*"]
```

Comment:
Perhaps call it path_filters instead of just path.

Reply:
I wanted to copy the `path` parameter from the [Logstash file input plugin](https://www.elastic.co/guide/en/logstash/current/plugins-inputs-file.html#plugins-inputs-file-path), but I can change that.